test/config.jl

```diff
@@ -421,4 +421,38 @@
             rm(sandbox_dir; recursive = true)
         end
     end
+
+    @testset "ignored" begin
+        unformatted_text = "( )"
+        sandbox_dir = joinpath(tempdir(), "test_ignored_config")
+        tobeignored = (
+            "b.jl",
+            "ignored_directory/a.jl",
+            "other_directory/ignored_directory/a.jl",
+            "other_directory/directory/b.jl",
+            "third_directory/a.jl",
+            "third_directory/ignored_directory/a.jl",
+        )
+        nottobeignored = (
+            "a.jl",
+            "other_directory/a.jl",
+            "other_directory/directory/a.jl",
+            "third_directory/b.jl",
+            "third_directory/ignored_directory/b.jl",
+        )
+        try
+            cp("files/ignore", sandbox_dir)
+            @test format(sandbox_dir) == false
```

Comment:
jw if this is run again the output the true correct?

Reply:
Yes. I should add a second run.

Reply:
other than that this looks really good, appreciate you working on this

Reply:
I'm having fun. Thank you for writing a good package!